core/openapi/schemas/contact.yaml

```diff
@@ -44,12 +44,13 @@ properties:
     description: Telephone numbers at which contact can be made.
     items:
       type: object
-      required: value
+      required:
+        - value
       properties:
         value:
           type: string
           description: The value is the phone number itself.
-          pattern: "^+[1-9]{1}[0-9]{3,14}$"
+          pattern: "^\\+[1-9]{1}[0-9]{3,14}$"
```

Comment:
Weird. The change is correct, but we had another PR recently that fixed this and I'm wondering why the change got lost...